README.md

```diff
@@ -1,2 +1,42 @@
-# Hello-Microverse
-first linter project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+  Hello Microverse project
+
+> In this porject we used linters and added html and css files.
```

Comment:
[OPTIONAL]
- You seem to have a typo here. Instead of 'project', you have 'porject'. Please ensure to prevent typos to ensure clear communication to a user 👍